shell/app/modules/runtime/pages/overview/components/index.tsx

```diff
@@ -192,7 +201,9 @@ const RuntimeOverView = () => {
           <ErrorBoundary>
             <IF check={!isEmpty(endpoints)}>
               <div className="overview-body-block">
-                <div className="overview-body-title">{i18n.t('runtime:endpoint')}</div>
+                <div className="overview-body-title">
+                  {runtimeType === 'job' ? i18n.t('task') : i18n.t('runtime:endpoint')}
```

Comment:
Endpoints is service which have domain, but task can not have domain, so endpoints is always empty if runtimeType is task.